plugins/github-issues/api-report.md

```diff
@@ -5,9 +5,49 @@
 ```ts
 /// <reference types="react" />
 
+import { ApiRef } from '@backstage/core-plugin-api';
 import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { ConfigApi } from '@backstage/core-plugin-api';
+import { ErrorApi } from '@backstage/core-plugin-api';
+import { OAuthApi } from '@backstage/core-plugin-api';
 import { RouteRef } from '@backstage/core-plugin-api';
 
+// @public (undocumented)
+export type Assignee = {
+  avatarUrl: string;
+  login: string;
+};
+
+// @public (undocumented)
+export type EdgesWithNodes<T> = {
+  edges: Array<{
+    node: T;
+  }>;
+};
+
+// @public (undocumented)
+export type GitHubIssuesApi = ReturnType<typeof gitHubIssuesApi>;
+
+// @public (undocumented)
+export const gitHubIssuesApi: (
+  githubAuthApi: OAuthApi,
+  configApi: ConfigApi,
+  errorApi: ErrorApi,
+) => {
+  fetchIssuesByRepoFromGitHub: (
+    repos: Array<string>,
+    itemsPerRepo: number,
+  ) => Promise<IssuesByRepo>;
+};
+
+// @public (undocumented)
+export const gitHubIssuesApiRef: ApiRef<{
```

Comment:
Are there currently a good use case for exporting this API alongside the other types? If not I would advice that we keep these exports internal until they're needed externally :) If they are needed externally however there are probably a few changes that needs to be done. I would rename assignee to `gitHubIssuesAssignee` to make it clear when auto importing that it belongs to this plugin. EdgesWithNodes is also a very generic generic and should probably be a proper type not to leak too much of the underlying graphql implementation.

Reply:
Great, let's try to make it internal then. Feel free to raise all questions and problems in this PR :)

Reply:
Turns out it works without issues. Here is a commit with the update: [88752ee](https://github.com/backstage/backstage/pull/13059/commits/88752eed7e9dda89fc284e0758b18f91f26bef61)

Reply:
Hi @jhaals 👋 I have implemented the requested changes.